src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThread.test.js

```diff
@@ -1,22 +1,74 @@
 // @flow
 import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
-
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import ActivityThread from '../ActivityThread.js';
+import replies from '../../../../../__mocks__/replies';
 
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
 describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread', () => {
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<ActivityThread {...props}>Test</ActivityThread>);
+    const defaultProps = {
+        replies,
+        totalReplyCount: 2,
+        hasReplies: true,
+    };
+
+    function customRender(ui, { locale = 'en', ...renderOptions } = {}) {
+        function Wrapper({ children }: { children: React.Component }) {
+            return <IntlProvider locale={locale}>{children}</IntlProvider>;
```

Comment:
How do we handle this wrapper boilerplate in other RTL tests in BUIE? We have helper methods to handle a lot of this for us in other codebases.

Reply:
The only usage I have found in `ActivitySidebarFilter.test.js.` It seems like we do not have any boilerplate for it in BUIE.